commands/hubdb/fetch.ts

```diff
@@ -1,37 +1,40 @@
-// @ts-nocheck
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const { logError } = require('../../lib/errorHandlers/index');
-const { downloadHubDbTable } = require('@hubspot/local-dev-lib/hubdb');
-const {
-  selectHubDBTablePrompt,
-} = require('../../lib/prompts/selectHubDBTablePrompt');
-const { trackCommandUsage } = require('../../lib/usageTracking');
-
-const {
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { logError } from '../../lib/errorHandlers/index';
+import { downloadHubDbTable } from '@hubspot/local-dev-lib/hubdb';
+import { selectHubDBTablePrompt } from '../../lib/prompts/selectHubDBTablePrompt';
+import { trackCommandUsage } from '../../lib/usageTracking';
+import {
   addConfigOptions,
   addAccountOptions,
   addUseEnvironmentOptions,
-} = require('../../lib/commonOpts');
-const { i18n } = require('../../lib/lang');
+} from '../../lib/commonOpts';
+import { i18n } from '../../lib/lang';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
 
 const i18nKey = 'commands.hubdb.subcommands.fetch';
 
-exports.command = 'fetch [table-id] [dest]';
-exports.describe = i18n(`${i18nKey}.describe`);
+export const command = 'fetch [table-id] [dest]';
+export const describe = i18n(`${i18nKey}.describe`);
+
+type HubdbFetchArgs = CommonArgs &
+  ConfigArgs & { tableId?: number; dest?: string };
```

Comment:
Do we have a system to determine which args are optional and which ones are required? It looks like we have a mix. I think I had made them required when I did the types for the account commands, but I think it actually makes more sense to make them optional as long as we don't define a default value for the option down in the builder. Maybe I'll fix the account commands to follow this pattern.

It looks like the hubdb/create args could use `path?:` too

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1372/commits/a33d8a0132a153f1dab2785447a92e7c312e4375